maat-scheduled-tasks/src/main/java/uk/gov/justice/laa/maat/scheduled/tasks/entity/ApplicantEntity.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.justice.laa.maat.scheduled.tasks.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "APPLICANTS", schema = "TOGDATA")
+public final class ApplicantEntity {
```

Comment:
There are a few fields in here that we don't need, such as `specialInvestigation`. The entity you need does already exist though, albeit perhaps not obviously named: _BillingApplicantEntity_.

This is what needs copying over as-is 🙂